cfme/tests/infrastructure/test_vmware_provider.py

```diff
@@ -502,6 +509,7 @@ def test_rebuilt_vcenter_duplicate_hosts(appliance, provider):
     assert result.success, "SSH Command result was unsuccessful: {}".format(result)
     logger.info('output of rails command: %s', result.output)
     provider.refresh_provider_relationships()
+    wait_for(provider.is_refreshed, func_kwargs={"refresh_delta": 120}, num_sec=300, delay=30)
```

Comment:
@kedark3 In my opinion, this should either be built in to `refresh_provider_relationships()` or a function should be available in `cfme.common.provider.py` that allows for this functionality (e.g. `provider.wait_for_provider_refresh(**kwargs)`). What do you think?

Reply:
PR LGTM though :+1:

Reply:
@john-dupuy I agree but this would be out of scope for this PR